services/subscriptions/handlers.go

```diff
@@ -9,11 +10,26 @@ import (
 // GetSubscriptionsHandler is the GET method handler for /subscriptions
 func (svc *SubscriptionService) GetSubscriptionsHandler() echo.HandlerFunc {
 	return func(c echo.Context) error {
-		response := SubscriptionResponse{
-			Value: svc.Subscriptions,
+		subsString, err := svc.Store.Get("subscriptions")
+		if err != nil {
+			panic(err)
 		}
 
-		return c.JSON(http.StatusOK, response)
+		// If something is stored in the db
+		if subsString != "" {
```

Comment:
Do we want to have error handling here instead of having to empty check every database check?